src/main/java/qna/domain/Question.java

```diff
@@ -1,12 +1,33 @@
 package qna.domain;
 
-public class Question {
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "question")
+public class Question extends BaseTimeEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+
+    @Column(name = "title", nullable = false)
```

Comment:
DDL을 보시면 title의 length 에 대한 내용이 있는데요
요구사항을 한번 확인 해주세요! 😄 
```suggestion
    @Column(name = "title", nullable = false, length = 100)
```